src/components/presentation/nodeDetails.ts

```diff
@@ -21,34 +25,33 @@ import CountSensorSchema from "../models/readings/CountSensorSchema";
 import TotalSensorSchema from "../models/readings/TotalSensorSchema";
 
 // eslint-disable-next-line import/prefer-default-export
-export function getSensorDetailsPresentation(
-  sensor?: Sensor,
+export function getNodeDetailsPresentation(
+  node?: Node,
   gateway?: Gateway,
   readings?: SensorReading<unknown>[]
-): SensorDetailViewModel {
+): NodeDetailViewModel {
   return {
     gateway: {
       serialNumber: gateway?.serialNumber || GATEWAY_MESSAGE.NO_SERIAL_NUMBER,
     },
-    sensor: sensor
+    node: node
       ? {
-          name: sensor.name || SENSOR_MESSAGE.NO_NAME,
-          lastActivity: getFormattedLastSeen(sensor.lastActivity),
-          location: sensor?.location || SENSOR_MESSAGE.NO_LOCATION,
+          name: node.name || NODE_MESSSAGE.NO_NAME,
+          lastActivity: getFormattedLastSeen(node.lastActivity),
+          location: node?.location || NODE_MESSSAGE.NO_LOCATION,
           temperature:
-            getFormattedTemperatureData(sensor.temperature) ||
+            getFormattedTemperatureData(node.temperature) ||
             SENSOR_MESSAGE.NO_TEMPERATURE,
```

Comment:
All of these `SENSOR_MESSAGE` constants should be `NODE_MESSAGE` too, correct?

Reply:
This one is tough because these messages kind of are from legit sensors, which is why I left them.